gazebo_ros/scripts/spawn_model

```diff
@@ -243,6 +248,9 @@ class SpawnModel():
           print "Error: user specified param or filename is an empty string"
           sys.exit(0)
 
+        if self.package_to_model:
+            model_xml = re.sub("<\s*mesh\s+filename\s*=\s*\"package:/?/?","<mesh filename=\"model://", model_xml)
```

Comment:
this only works for attributes with double quotes `"`. I think it should be updated to handle single quotes `'` as well